src/routes/TopicPage/index.jsx

```diff
@@ -0,0 +1,10 @@
+import { useParams } from 'react-router-dom';
+import { RenderArticles } from '../../components/RenderArticles/RenderArticles';
+
+export const TopicPage = () => {
```

Comment:
not sure i entirely see the point of having topic page and render articles as separate components...